src/docs/development/data-and-backend/google-apis.md

```diff
@@ -65,7 +66,19 @@ when authenticating the user.
 import 'package:googleapis/youtube/v3.dart';
 ```
 
-## 2. Authenticate the user with the required scopes
+## 2. Enable the API
+
+To use Google APIs you must have a Google account and a Google project. You also
+need to enable your desired API.
+
+In this example, you'd enable
+[YouTube Data API v3](https://console.cloud.google.com/apis/api/youtube.googleapis.com).
+
+Follow the
+[getting started instructions](https://cloud.google.com/apis/docs/getting-started)
+to learn more.
```

Comment:
Are we following instructions or just learning? Maybe:

```suggestion
For details, see the
[getting started instructions](https://cloud.google.com/apis/docs/getting-started).
```

Reply:
Following instructions. Just didn't want to duplicate a bunch of content that may evolve.